cmd/porter/mixins.go

```diff
@@ -197,3 +198,32 @@ func BuildMixinFeedTemplateCommand(p *porter.Porter) *cobra.Command {
 	}
 	return cmd
 }
+
+func buildMixinsCreateCommand(p *porter.Porter) *cobra.Command {
+	opts := porter.MixinsCreateOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "create NAME --author YOURNAME [--dir /path/to/mixin/dir]",
+		Short: "Create a new mixin project based on the getporter/skeletor repository",
+		Long: `Create a new mixin project based on the getporter/skeletor repository.
+
+The first argument is the name of the mixin to create and is required.
+A flag of --author to declare the author of the mixin is also a required input.
+You can also specify where to put mixin directory. It will default to the current directory.`,
+		Example: ` porter mixin create MyMixin --author MyName
+		porter mixin create MyMixin --author MyName --dir path/to/mymixin
```

Comment:
```suggestion
		porter mixin create MyMixin --author "My Name" --username myuser --dir path/to/mymixin
```

Let's show them how to use a full name with spaces, and then include a new flag for the username (more on that below).